test/fixtures/generator/bwx_spec.json

```diff
@@ -13,7 +13,20 @@
     "configuration": "darwin_x86_64-fastbuild-ST-1b9bd654f600",
     "custom_xcode_schemes": [
         {
-            "build_action": null,
+            "build_action": {
+                "targets": [
+                    {
+                        "build_for": {
+                            "analyzing": "unspecified",
+                            "archiving": "enabled",
+                            "profiling": "unspecified",
+                            "running": "unspecified",
+                            "testing": "unspecified"
```

Comment:
Future enhancement: we could not encode when `unspecified` and adjust the Decodeable conformance to default it. Will save some space in these files (which is a goal of #237).